pkg/instancetype/upgrade/upgrade_test.go

```diff
@@ -145,9 +145,6 @@ var _ = Describe("ControllerRevision upgrades", func() {
 
 		Expect(sanityUpgrade(vm)).To(Succeed())
 
-		vm, err = virtClient.VirtualMachine(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
```

Comment:
This seems wrong, below you are asserting on the vm but the variable is from the `Create`. Where does the Update gets called for the new controller ref?

Reply:
`sanityUpgrade` upgrades the VM in place.

See https://github.com/kubevirt/kubevirt/blob/1d2c578e3ee5c19df36a6f32e711c4086f46e2f4/pkg/instancetype/upgrade/handler.go#L56